### PR TITLE
chore(j-s): Add tail to all sms and emails with a link to RVG

### PR DESCRIPTION
**Pull request by oddsson**
# Add tail to all sms and emails with a link to RVG

Asana

## What

Add tail to all sms and emails with a link to RVG.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by unakb — APPROVED** [review]

**Comment by unakb** [comment]
Some tests might break, but looks good !

File: apps/judicial-system/backend/src/app/modules/notification/notification.service.ts [hunk 1]
```diff
@@ -853,6 +869,8 @@ export class NotificationService {
           }),
       defenderName ?? '',
       defenderEmail ?? '',
+      [],
```
Comment (unakb) [inline apps/judicial-system/backend/src/app/modules/notification/notification.service.ts:872]: Is there a reason this Attachment argument is a empty array while the others are undefined ?

File: apps/judicial-system/backend/src/app/modules/notification/notification.service.ts [hunk 2]
```diff
@@ -1492,6 +1517,8 @@ export class NotificationService {
       html,
       theCase.registrar.name,
       theCase.registrar.email,
+      [],
```
Comment (unakb) [inline apps/judicial-system/backend/src/app/modules/notification/notification.service.ts:1520]: Same here

**Review by unakb — APPROVED** [review]
Just one question otherwise nice

File: apps/judicial-system/backend/src/app/modules/notification/notification.service.ts (hunk 1 above)
Comment (oddsson) [reply apps/judicial-system/backend/src/app/modules/notification/notification.service.ts]: Nope, just a mistake..

**Pull request merged**